src/utils/alerts.js

```diff
@@ -20,7 +24,7 @@ export const alertPopup = (title, html, confirmLabel, confirmAction, cancelLabel
             }
         }
     ).then((result) => {
-        if (result.isConfirmed) {
+        if (result.value) {
             confirmAction();
         } else if (result.isDenied) {
```

Comment:
@tomrndom please lets remove this else if.
Dismissing results in {dismiss: 'cancel'}, not {isDenied: true}.